sites/geohub/src/lib/RasterTileData.ts

```diff
@@ -3,21 +3,32 @@ import {
 	getActiveBandIndex,
 	getDefaltLayerStyle,
 	getDefaltLayerStyleForStac,
+	getDefaultLayerStyleForStacProducts,
 	getFirstSymbolLayerId
 } from './helper';
 import type { RasterTileMetadata, DatasetFeature, LayerCreationInfo } from './types';
 import type { Map } from 'maplibre-gl';
 
 export class RasterTileData {
 	private feature: DatasetFeature;
+	private product?: string;
 
-	constructor(feature: DatasetFeature) {
+	constructor(feature: DatasetFeature, product?: string) {
```

Comment:
As I added comment before, you can remove this variable and get info from feature.properties.tag within constructor.